README.md

```diff
@@ -1,17 +1,22 @@
 # Byndyusoft.Data.Relational.QueryBuilder [![Nuget](https://img.shields.io/nuget/v/Byndyusoft.Data.Relational.QueryBuilder.svg)](https://www.nuget.org/packages/Byndyusoft.Data.Relational.QueryBuilder/) [![Downloads](https://img.shields.io/nuget/dt/Byndyusoft.Data.Relational.QueryBuilder.svg)](https://www.nuget.org/packages/Byndyusoft.Data.Relational.QueryBuilder/)
 
-Библиотека для постороения запросов.
+Library for query building.
 
-Основная концепция библиотеки заключается в том, чтобы в запросах не использовать строковые константы для названий колонок. Так можно бьть увереннее и спокойнее рефакторить код. К тому же не нужно будет проверять постоянно, правильно ли написаны типовые (и не очень) запросы.
+The main concept of the library is to avoid using string constants for column names in queries. This allows for more confident and peaceful code refactoring. Additionally, there is no need to constantly check if the standard (or non-standard) queries are written correctly.
```

Comment:
This allows for more confident and peaceful code refactoring =>
You will be more confident in your code and less stressful while refactoring it.

Reply:
there is no need to constantly check if =>
you won't need to constantly check if

Reply:
I do not quite grasp the need to use "you" in those phrases. To be more personal with a potential library consumer? It goes without saying "allows for" has a different meaning than it was obviously intended to convey. However, I would recommend something like "This would allows to be more confident and peaceful while performing code refactoring. Additionally, this can reduce the need to check if queries are written correctly."

Reply:
This would allows => This would allow